### DISCUSSION TITLE
Unable to edit Frigate config file from Frigate GUI in docker container.

### DISCUSSION THREAD

**Discussion by echo4019** [blakeblackshear/frigate#8523] [Ask A Question]
Hi, 

I have managed to get Frigate installed on a Proxmox LXC and have access to the Frigate GUI via IP:5000

When I try to edit the config within the GUI it gives me the following error: "Could not write config file, be sure that Frigate has write permission on the config file."

I have found out that this is being caused by the compose file having the characters :ro next to the config file location

I have navigated to nano docker-compose.yaml in the container console and removed ":ro" and saved the file and restarted but the error message remains. 

I am quite a novice to Proxmox and Linux commands so there is a good chance I am doing something wrong.

Any help is appreciated.

Thanks.

**Comment by NickM-27** [comment]
what did you do to restart, most likely the new compose config has not been applied as restarting does not do that

**Reply by echo4019** [reply]
I simply saved the file by pressing Control + S and clicked restart on the docker instance. 

What is the proper process to apply changes?

Thanks.

**Reply by NickM-27** [reply]
restarting doesn't apply the new docker compose file, you still have to run `docker compose down` then `docker compose up -d`

**Reply by echo4019** [reply]
Ah that makes sense, so after I save I then go back into the console and type docker "compose down" then docker "compose up -d"?

Do I then need to restart docker?

Thanks.

**Reply by NickM-27** [reply]
That is restarting docker

**Reply by echo4019** [reply]
Got it. Thanks for your help. I'll give this a shot when I get home. 

On a side note, do you know of any videos I could watch that could give me a basic understanding on these docker commands? 

Thanks.

**Reply by echo4019** [reply]
So I have ran the command "docker compose down" then "docker compose up -d" but I am getting the following error message: 

root@docker-frigate:/opt# docker compose up -d
[+] Building 0.0s (0/0)                                                                                                                                       docker:default
[+] Running 0/0
 ⠋ Container frigate  Creating                                                                                                                                          0.0s 
Error response from daemon: Conflict. The container name "/frigate" is already in use by container "65bb4cae6e88a63e0ab4e90616d1ad3e64e700dc4c327d8f02b1ae18281fae17". You have to remove (or rename) that container to be able to reuse that name.

Not sure what I'm doing wrong. 

Thanks
